source/index.ts

```diff
@@ -94,6 +94,12 @@ Object.defineProperties(ow, {
 
 			test(value, labelOrPredicate as string, predicate as BasePredicate<T>);
 		}
+	},
+	characterSet: {
+		value: {
+			base64: 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=',
```

Comment:
I would add a doc comment here with a warning that it only validates whether a string has valid `base64` characters, not whether it's a valid `base64` string. Same with `base58`.